### ISSUE TITLE
The SQLite3 version on Windows is outdated

### ISSUE THREAD

**Issue by Hubro** [sparklemotion/sqlite3-ruby#134]
I've been debugging a weird performance issue for about a month. On Linux, all my queries run super fast through the SQLite3 CLI and sqlite3-ruby. On Window, all my queries run super fast through the SQLite3 CLI, but take thousands of times longer to complete through sqlite3-ruby. This was confusing, because changing my query in any way (removing the sort portion, removing `INDEXED BY`, replacing `GLOB` with `=` etc.) would fix the issue.

Long story short, turns out the SQLite3 version in the windows prebuilt sqlite3-ruby gem is outdated (3.7.17). I built the gem manually with sqlite3 version 3.8.5 and the performance issue disappeared. This won't help the users of my gem until the prebuilt windows gem is updated though, hence this issue.

Thanks.
